### PR TITLE
Feature/swizzler

### PR DESCRIPTION
**Pull request by vadymmarkov**
@zenangst @RamonGilabert 
Convenience method for method swizzling:
```swift
// replaced with "swizzled_method"
Swizzler.swizzleMethod("method", cls: ClassName.self)

// replaced with "sugar_method"
Swizzler.swizzleMethod("method", cls: ClassName.self, prefix: "sugar")
```

### CODE REVIEW

File: Source/Shared/Swizzler.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+struct Swizzler {
+
+  enum Kind {
+    case Instance
+    case Class
+  }
+
+  static func swizzleMethod(method: String, cls: AnyClass!, prefix: String = "swizzled", kind: Kind = .Instance) {
```
Comment (zenangst) [inline Source/Shared/Swizzler.swift]: Could we rename it to `swizzle`? That you are swizzling a method is a part of the signature in the labels?

Comment (vadymmarkov) [inline Source/Shared/Swizzler.swift]: @zenangst 
Like `swizzle(method method: String...` or just `swizzle(method: String..`?

Comment (zenangst) [inline Source/Shared/Swizzler.swift]: @vadymmarkov which do you prefer? :grin:

Comment (vadymmarkov) [inline Source/Shared/Swizzler.swift]: I think second one is enough, it's obvious that you wanna swizzle a method.

**Comment by zenangst** [comment]
@vadymmarkov :tada: :cake:

**Pull request merged**